### DISCUSSION TITLE
IntellJ PUSH 오류입니다

### DISCUSSION THREAD

**Discussion by mihyeon714** [T-lab-java-king/tlab-java-core#38] [Q&A] [answered]
1.로컬 branch 생성 : origin/feature/18
2.파일 생성 test.md
3. commit 
**4. push > 토큰 생성 > 오류발생**

어떻게 진행하면 될까요~?

![image](https://github.com/T-lab-java-king/tlab-java-core/assets/6947102/e03d0a75-fe55-4419-82c1-6cecbc1befbc)

**Comment by choichanhyeok** [comment, marked as answer]
와 대박 ..
주말에도 열심히 하시고 벌써 PR까지 시도해보시는 모습 멋지십니다!

확인 해보니 멤버 기본 권한 설정이 '읽기'로 되어 있어서 푸쉬가 안되셨던 거 같습니다! 혹시 권한을 변경했는데 시간 괜찮으실 때 다시 진행해보실 수 있으실까요?

**Reply by mihyeon714** [reply]
넵 감사합니다 
다시 진행해보고 문제있으면 다시 문의드리겠습니다~!
